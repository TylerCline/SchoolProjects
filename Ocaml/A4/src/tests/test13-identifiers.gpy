let abc123___ABC123ABC999__ = 3
in abc123___ABC123ABC999__