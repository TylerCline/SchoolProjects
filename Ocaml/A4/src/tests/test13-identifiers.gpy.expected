
((let abc123___ABC123ABC999__ = (3 : int) in
(abc123___ABC123ABC999__ : int)) : int)

